src/features/ModelPlan/ModelToOperations/_components/ITSystemsTable/index.tsx

```diff
@@ -509,6 +516,13 @@ const ITSystemsTable = ({
     }
   }, [isPrintPDF, setPageSize, initPageSize]);
 
+  useEffect(() => {
+    if (params.get('page')) {
+      const pageNum = parseInt(params.get('page') || '0', 10);
+      gotoPage(pageNum - 1);
+    }
+  }, [params, gotoPage]);
```

Comment:
QA # 93 still exist when I tested locally. I did some console logging and it seems to me the issue is despite correct pageNum being passed down, the `state.pageIndex` is resetting back to the initial state.

Reply:
I'm going to hold off on 93 for now.  Talked with steven a bit on Friday, and we are going to tackle a few of these post launch, thanks!